lib/types.js

```diff
@@ -111,6 +111,7 @@ function Type(schema, opts) {
 Type.forSchema = function (schema, opts) {
   opts = opts || {};
   opts.registry = opts.registry || {};
+  opts.noStackTraces = opts.noStackTraces !== undefined ? opts.noStackTraces : false;
```

Comment:
I usually defer until later, except where recursion makes it easier to do it upfront. In this case I'd simply pass the options through, not mentioning it here (see below).